emission/core/wrapper/common_trip.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import emission.core.wrapper.wrapperbase as ecwb
+
+class CommonTrip(ecwb.WrapperBase):
+    props = {"start_loc" : ecwb.WrapperBase.Access.WORM, # _id of common_place of starting point
```

Comment:
In trip and place, I've used XXX_loc for the geojson. So if we followed that convention, then this would be start_point or start_common_place, and start_loc would be an additional field that would store the geojson representation of the point to support mongo geojson queries.